docs/07-audits/index.md

```diff
@@ -6,9 +6,11 @@ description: The following details audit reports related to the various Sygma re
 ---
 
 :::info
-The following details audit reports related to the various Sygma repositories.
+The following table details the security audit reviews that have been performed on various Sygma [code repositories](https://github.com/sygmaprotocol).
 :::
 
-import DocCardList from '@theme/DocCardList';
+| Date | Auditor | Audited Repos | Link | Status |
+| ---- |-------------| ------ | ------- | :----: |
+| Sept 6, 2022 | [KALOS (prev. HAECHI AUDIT)](https://www.kalos.xyz/) | [sygma-solidity v1.0](https://github.com/sygmaprotocol/sygma-solidity/releases/tag/v1.0.0), [sygma-relayer v1.0](https://github.com/sygmaprotocol/sygma-relayer/releases/tag/v1.0.0) | [Link](/assets/[HAECHI%20AUDIT]%20Sygma%20Audit%20Report%20v1.1.pdf) | ✅ |
+| May 18, 2023 | [Least Authority](https://leastauthority.com/) | [sygma-solidity v2.5](https://github.com/sygmaprotocol/sygma-solidity/releases/tag/v2.3.0), [sygma-relayer v1.5](https://github.com/sygmaprotocol/sygma-relayer/releases/tag/v1.5.0), [sygma-substrate-pallets v0.2](https://github.com/sygmaprotocol/sygma-substrate-pallets/releases/tag/sygma-bridge-v0.2.0)| [Link](https://docs.buildwithsygma.com/assets/files/least-authority-17-05-2023-a4c57b963354b382b6bea3c94cd30fc1.pdf) | ✅ |
```

Comment:
Release versions inconsistent for sygma-solidity v2.5 and the tag which is for v2.3

Reply:
ah good catch, i was following @MakMuftic's versioning on PROD at [https://docs.buildwithsygma.com/audits/v1.5/](https://docs.buildwithsygma.com/audits/v1.5/)

Reply:
also we should scrap this because base branch is `develop`. i will open a separate PR to update @MakMuftic's versioning on PROD